### PR TITLE
feat(scen): Add descriptions to each scenario

### PR DESCRIPTION
**Pull request by s-irvine**
Descriptions have been added to each scenario and are outputted on `simulator scenario list`. The format of `simulator scenario list` has also been updated to better fit with outputs that are over a line long.

The `rbac-shoot-and-scoot` scenario received a minor update to make it more unique.

### CODE REVIEW

File: cmd/scenario.go [hunk 1]
```diff
@@ -25,7 +25,10 @@ func newScenarioListCommand(logger *zap.SugaredLogger) *cobra.Command {
 
 			fmt.Println("Available scenarios:")
 			for _, s := range manifest.Scenarios {
-				fmt.Println("ID: " + s.Id + ", Name: " + s.DisplayName)
+				fmt.Println("")
```
Comment (abdullahgarcia) [inline cmd/scenario.go]: Sam, I believe this change will print a hard-to-read list as it is jumping a line per attribute.

File: simulation-scripts/scenario/rbac-shoot-and-scoot/challenge.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
-Scenario: A developer has informed you, they are able to make Kubernetes API calls from an Ubuntu pod although it only belongs to the default service account.
+Scenario: A developer has informed you, they are able to make Kubernetes API calls from an Ubuntu pod to get a secret although its service account token is unmounted.
 
 Starting Point: ubuntu pod in the rbac-shoot-and-scoot namespace
 
 Difficulty: Easy
 
-Task 1: Can you replicate this issue.
+Task 1: Can you replicate this issue by finding the secret value in the rbac-shoot-and-scoot namespace.
```
Comment (abdullahgarcia) [inline simulation-scripts/scenario/rbac-shoot-and-scoot/challenge.txt]: "namespace?"

File: simulation-scripts/scenario/rbac-shoot-and-scoot/tasks.yaml [hunk 3]
```diff
@@ -13,6 +13,13 @@ tasks:
       - text: >-
           Can you hit the API from inside the pod? What is the DNS resolvable
           name?
+      - text: What do we get from `printenv`?
+      - text: That looks like the IP and port for the API server!
+      - text: I reckon I can curl that endpoint if i use HTTPS and the -k option
```
Comment (abdullahgarcia) [inline simulation-scripts/scenario/rbac-shoot-and-scoot/tasks.yaml]: "if I use"

File: simulation-scripts/scenario/rbac-shoot-and-scoot/tasks.yaml [hunk 4]
```diff
@@ -13,6 +13,13 @@ tasks:
...
+      - text: How do I navigate the HTTP api to find a secret?
```
Comment (abdullahgarcia) [inline simulation-scripts/scenario/rbac-shoot-and-scoot/tasks.yaml]: "HTTP API"

File: simulation-scripts/scenarios.yaml [hunk 5]
```diff
@@ -5,91 +5,114 @@ scenarios:
...
   - name: Master bait and bleed
     path: ./scenario/master-bait-and-bleed
     id: master-bait-and-bleed
     difficulty: Medium
+    description: "A pod can make calls it should not have permissions for, how serious is this?"
   - name: Master Encirclement
     path: ./scenario/master-encirclement
     id: master-encirclement
     difficulty: Medium
+    description: "A deployment should have been blocked by the cluster, how did it succeed"
```
Comment (abdullahgarcia) [inline simulation-scripts/scenarios.yaml]: "succeed?"

File: simulation-scripts/scenarios.yaml [hunk 6]
```diff
@@ -5,91 +5,114 @@ scenarios:
...
   - name: Network Hedgehog Defense
     path: ./scenario/network-hedgehog-defence
     id: network-hedgehog-defence
     difficulty: Medium
+    description: "You are running a postmortem on a suspected attack, could they have crossed a namespace?"
   - name: Network Swarming
     path: ./scenario/network-swarming
     id: network-swarming
     difficulty: Easy
+    description: "Some odd audit logs have surfaced from your database, did someone manage to bypass your api?"
```
Comment (abdullahgarcia) [inline simulation-scripts/scenarios.yaml]: "your API?"

File: cmd/scenario.go (hunk 1 above)
Comment (s-irvine) [reply cmd/scenario.go]: It looks like this:
```
Available scenarios:

Name: Container Ambush
Description: Your CI build agent is compromised, where can you get from here?
ID: container-ambush

Name: Container Defeat in Detail
Description: A minor DaemonSet misconfiguration has led to cluster compromise, how?
ID: container-defeat-in-detail

Name: Container Phalanx Formation
Description: Use kubectl to investigate a logging bug that may be leaking secrets.
ID: container-phalanx-formation
...
```
I found that moving to multiple lines per scenario without a clear delimiter (space or colour) is quite hard to read personally. I think this is a nice compromise.

File: simulation-scripts/scenario/rbac-shoot-and-scoot/challenge.txt (hunk 2 above)
Comment (s-irvine) [reply simulation-scripts/scenario/rbac-shoot-and-scoot/challenge.txt]: The secret value is in a secret object in the namespace

Comment (abdullahgarcia) [reply simulation-scripts/scenario/rbac-shoot-and-scoot/challenge.txt]: Apologies, I was trying to highlight the lack of the question mark.

File: simulation-scripts/scenarios.yaml (hunk 6 above)
Comment (s-irvine) [reply simulation-scripts/scenarios.yaml]: I wanted to draw a distinction between the kube api and the services moke api for this scenario

File: simulation-scripts/scenario/rbac-shoot-and-scoot/challenge.txt (hunk 2 above)
Comment (s-irvine) [reply simulation-scripts/scenario/rbac-shoot-and-scoot/challenge.txt]: Lol, will fix

File: simulation-scripts/scenarios.yaml (hunk 6 above)
Comment (s-irvine) [reply simulation-scripts/scenarios.yaml]: I see what you mean, have pushed a change

File: simulation-scripts/scenarios.yaml [hunk 7]
```diff
@@ -5,91 +5,114 @@ scenarios:
...
   - name: Network Hedgehog Defense
     path: ./scenario/network-hedgehog-defence
     id: network-hedgehog-defence
     difficulty: Medium
+    description: "You are running a postmortem on a suspected attack; could they have crossed a namespace?"
   - name: Network Swarming
     path: ./scenario/network-swarming
     id: network-swarming
     difficulty: Easy
+    description: "Some odd audit logs have surfaced from your database; did someone manage to bypass your api?"
```
Comment (abdullahgarcia) [inline simulation-scripts/scenarios.yaml]: The last word: API, should be in upper case I believe.

Comment (s-irvine) [reply simulation-scripts/scenarios.yaml]: I'd made the change, just hadn't pushed

**Comment by s-irvine** [comment]
fixes #130

**Pull request merged**